Documentation/Docs-CSS/main-section.css

```diff
@@ -5,7 +5,7 @@
 .component-section {
   width: 90vw;
   padding: 1rem;
-  margin: 1rem auto;
+  margin: 3rem auto;
   height: fit-content;
   background-color: var(--secondary);
   color: var(--primary);
```

Comment:
Nice play with the dark background. One suggestion - dark blue to the font color is a bit too jarring. 
You can also see the contrast ratio in the dev tools giving us a warning. See if you can play with some other colors for the fonts. 

![image](https://user-images.githubusercontent.com/38975782/151932355-9f93cda2-18af-4508-84ad-44d46b4cc401.png)

Reply:
Oh, right! I wanted to make all the text on dark background appear white but I think I forgot to remove the color property from the `component-section`. I will change it right away. Thanks for the review!